docs/pages/docs/cli/zipper.mdx

```diff
@@ -35,15 +35,15 @@ downstreams:
    - zipper-asia_pacific
      host: 2.2.2.2
      port: 9000
-     auth: 'token: <CREDENTIAL>'
+     credential: 'token: <CREDENTIAL>'
    - zipper-america
```

Comment:
yaml does not support such grammar, the `name` field should be added

Just like:
```yaml
downstreams:
   - name: zipper-asia_pacific
     host: 2.2.2.2
     port: 9000
     credential: 'token: <CREDENTIAL>'
```